providers/openfeature-go-feature-flag-provider/lib/openfeature/go-feature-flag/goff_api.rb

```diff
@@ -4,20 +4,20 @@
 require "net/http"
 require "json"
 require "faraday"
+require "faraday/net_http_persistent"
```

Comment:
question: do we still need to require `faraday` if we're using the `persistent` adapter?


```suggestion
require "faraday/net_http_persistent"
```